Formula/vlang.rb

```diff
@@ -0,0 +1,14 @@
+class Vlang < Formula
+  desc "V programming language"
+  homepage "https://vlang.io"
+  url "https://github.com/vlang/v/archive/0.1.18.tar.gz"
+  sha256 "3f3407a78aca7fc3b42a3fc1f1d2b9724c1e4c71fbd5d37ff12976cd2305cec1"
+
+  def install
+    system "make", "sudo v symlink"
```

Comment:
I'm pretty sure this does not do what you think it does.

Reply:
agree 👍